frontend/components/Logo.js

```diff
@@ -9,16 +9,17 @@ export class Logo extends React.Component {
         this.top = props.top;
         this.left = props.left;
         this.position_type = props.position;
+        this.logo_type = props.logo_type;
 
     }
 
     render() {
         return (
-            <div className="parent-bounding-box" style={{top: this.top+"%",
+            <div className={"parent-bounding-box-"+this.logo_type} style={{top: this.top+"%",
                                                          left: this.left+"%",
                                                          position:this.position_type}}>
-                <div className="bottom-right-red-rectangle"/>
-                <div className="top-left-blue-rectangle"/>
```

Comment:
Legend imported but never used. I'm also not seeing the logo anywhere? @qujohn314 can you add a screenshot of what you are seeing?

Thank you!

Reply:
![Capture3](https://user-images.githubusercontent.com/35878908/178777987-2b0418f4-0db9-4c9a-a726-7afc96252807.PNG)


![Capture](https://user-images.githubusercontent.com/35878908/178777495-07952651-ebca-47a5-89b7-892339b7d2f2.PNG)

Here's where it's used right now. I got rid of the side logo appearing in the bottom right corner since it was scrapped from the final design. The logo can change based on the screen size.

![Capture2](https://user-images.githubusercontent.com/35878908/178777847-353f52b5-1081-4a2d-aed0-0ba8ced86a49.PNG)